srml/contract/src/wasm/runtime.rs

```diff
@@ -628,4 +628,16 @@ define_env!(Env, <E: Ext>,
 
 		Ok(())
 	},
+
+	// Prints utf8 encoded string from the data buffer.
+	// MUST only be called from contracts deployed to `--dev` chains, NOT for use in production chains.
+	// This function may be removed in the future and superseded by "Diagnostic Runtimes":
```

Comment:
Not sure if we actually want to mention the exact solution. I can imagine that the #2082 can be superseeded by anything else, but this line will still stay here.

Also, I'd say that this function is only available on the "--dev" chains, not "must only be called"